### PR TITLE
Added method to return data url

### PR DESCRIPTION
**Pull request by akhilrex**
Hi

I have added a simple method that returns the data url that can be returned directly in the img tag. It just exposes the default functionality of canvas.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -119,6 +119,22 @@ class ChartjsNode {
             }, imageType);
         });
     }
+     /**
+     * Returns image in the form of Data Url
+     *
+     * @param {String} imageType The image type name. Valid values are image/png image/jpeg
+     * @returns {Promise} A promise that resolves when the image is received in the form of data url
+     */
+        getImageDataUrl(imageType) {
```
Comment (sedouard) [inline index.js]: NIT: Tab this back to line up with the other methods.

**Comment by sedouard** [comment]
Thanks @akhilrex! This looks awesome 👍 . Just one tiny formatting issue. If you can just fix it real quick I'll have it merged and published today.

Cheers! 🍻

**Comment by akhilrex** [comment]
Done!!

**Comment by sedouard** [comment]
Thanks!

**Comment by sedouard** [comment]
[`chartjs-node@1.3.0`](https://github.com/vmpowerio/chartjs-node/releases/tag/1.3.0) published has this change.

**Pull request merged**